HiP-FeatureToggle/Models/Entity/FeatureToFeatureGroupMapping.cs

```diff
@@ -0,0 +1,31 @@
+﻿namespace PaderbornUniversity.SILab.Hip.FeatureToggle.Models.Entity
+{
+    /// <remarks>
+    /// Since EF Core does not (yet) support many-to-many relationships, the relationship between features
+    /// and feature groups is modelled as two 1-to-many relationships. In other words, this class acts as
+    /// the join table.
+    ///
+    /// [Feature]----------------[FeatureToFeatureGroupMapping]-------------[FeatureGroup]
+    ///          1              n                              m           1
+    /// </remarks>
+    public class FeatureToFeatureGroupMapping
```

Comment:
While this would also work, we normally do it another way in our other projects and I think it should be consistent. We map relations using a dedicated *Map subclass in each entity class (so in this case you would maybe have a `FeatureMap` subclass in `Feature`). See for example here: https://github.com/HiP-App/HiP-CmsWebApi/blob/develop/src/Api/Models/Entity/Topic.cs

The *Map classes are then instantiated in the `DbContext`: https://github.com/HiP-App/HiP-CmsWebApi/blob/develop/src/Api/Data/CmsDbContext.cs